src/jit/gentree.h

```diff
@@ -4997,8 +5001,7 @@ struct GenTreeStoreInd : public GenTreeIndir
 
 struct GenTreeRetExpr : public GenTree
 {
-    GenTreePtr gtInlineCandidate;
-
+    GenTreePtr           gtInlineCandidate;
```

Comment:
While you're here, and since there seems to be consensus on this, you could change `GenTreePtr` to `GenTree*`

Reply:
Fixed.